cmd/provenanced/cmd/root.go

```diff
@@ -50,6 +50,10 @@ const (
 	EnvTypeFlag = "testnet"
 	// Flag used to indicate coin type.
 	CoinTypeFlag = "coin-type"
+	// CustomDenomFlag flag to take in custom denom, defaults to nhash if not passed in.
+	CustomDenomFlag = "custom-denom"
```

Comment:
imo denom is just `nhash` since before this story we always assumed it to be so(except i guess stake for sim-test etc).. my thought was vspn, fescoin etc are custom denom for the same code base 🤷

Reply:
Ah. Cool. Makes sense.